source/help/settings/account-settings.md

```diff
@@ -52,23 +52,34 @@ Sessions are created when you log in with your email and password to a new brows
 ## Notifications
 Settings to configure desktop notifications, desktop notification sounds, email notifications, and words that trigger mentions.
 
-#### Send Desktop Notifications
-Desktop notifications appear at the bottom right corner of your main monitor. The desktop notification preference you choose in *Account Settings* applies globally, but this preference is customizable for each channel from the channel name drop-down menu. Desktop notifications are available on Firefox, Safari, and Chrome.
+#### Desktop Notifications
+Desktop notifications appear at the bottom right corner of your main monitor when there is activity in Mattermost. 
```

Comment:
Not related to the changes, but is "Desktop notifications appear at the bottom right corner of your main monitor" true? Desktop notifications appear in the top left for me?